docs/extend/modular-resources/_index.md

```diff
@@ -77,13 +77,13 @@ If you are creating your own modular resource, follow these steps:
 
 {{% alert title="Modules vs. modular resources" color="tip" %}}
 
-A configured **module** can make one or more *modular resources* available for configuration.
+A configured *module* can make one or more *modular resources* available for configuration.
 
 {{% /alert %}}
 
 ### Code your module
 
-Code a module in the Go or Python programming languages with [Viam's SDKs](/program/apis/) that does the following:
+Use [Viam's Go or Python language SDKs](/program/apis/) to code a module that does either of the following:
```

Comment:
Can you actually do both in one module? If so
```suggestion
Use [Viam's Go or Python language SDKs](/program/apis/) to code a module that does either or both of the following:
```

Reply:
I think as is makes sense. You'll probably want to do either or most of the time?

Reply:
Maybe remove the word language? Feels a bit strange

Reply:
Okay! I was attempting to preserve the original sentence information but I agree it sounds strange :)
```suggestion
Use [Viam's Go or Python SDKs](/program/apis/) to code a module that does either of the following:
```